assets/src/amp-validation/copy-to-clipboard-buttons.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import Clipboard from 'clipboard';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getURLValidationTableRows } from './get-url-validation-table-rows';
+
+/**
+ * Success handler, called when data is copied to the clipboard.
+ *
+ * @param {Object} event
+ * @param {HTMLElement} event.trigger The element triggering the event.
+ */
+function onSuccess( { trigger } ) {
+	trigger.focus();
+
+	const newInnerText = __( 'Copied to clipboard', 'amp' );
+
+	// Exit if the user has already clicked the button and we are still within the
+	// 4000ms before the setTimeout callback runs.
+	if ( trigger.innerText === newInnerText ) {
+		return;
+	}
+
+	const originalText = trigger.innerText;
+	trigger.innerText = newInnerText;
+
+	setTimeout( () => {
+		if ( document.body.contains( trigger ) ) {
+			trigger.innerText = originalText;
+		}
+	}, 4000 );
+}
+
+/**
+ * Sets up the "Copy to clipboard" buttons on the URL validation screen.
+ */
+export function handleCopyToClipboardButtons() {
+	const clipboards = [];
+
+	// eslint-disable-next-line no-new
+	clipboards.push( new Clipboard( 'button.single-url-detail-copy', {
+		text: ( btn ) => {
+			const json = JSON.parse( btn.getAttribute( 'data-error-json' ) );
+			const statusSelect = btn.closest( 'tr' ).querySelector( '.amp-validation-error-status' );
+			json.status = statusSelect.options[ statusSelect.selectedIndex ].text;
+
+			return JSON.stringify( json, null, '\t' );
```

Comment:
I don't think we actually need to capture the `status` here because we already have the DB-_saved_ status in the `data-error-json` already. If we do include it, then it should be put into a separate field to not override `status` (or rather `removed` and `reviewed` per above). 

```suggestion
			return btn.getAttribute( 'data-error-json' );
```